### ISSUE TITLE
convert-commonjs-to-esm doesn't work

### ISSUE THREAD

**Issue by EvgenyOrekhov** [coderaiser/putout#57]
I enabled `convert-commonjs-to-esm/require` and `convert-commonjs-to-esm/exports` here: https://github.com/EvgenyOrekhov/eslint-config-hardcore/commit/90dc0972dc57e2ce5a3ac5183ec75d3b5f98bc22.

Now I expect it to complain about `require()` calls in [`test.js`](https://github.com/EvgenyOrekhov/eslint-config-hardcore/blob/master/test.js), but it doesn't.

What did I do wrong?

**Comment by coderaiser** [comment]
As I see, your [package.json](https://github.com/EvgenyOrekhov/eslint-config-hardcore/blob/master/package.json) doesn't have `type` field, so `ESM` `test.js` would not work any way (unless you rename it to test[.mjs](https://nodejs.org/api/packages.html#packages_package_json_and_file_extensions)). Also you `requiring` json files supported only with `nodejs` flag [--experimental-json-modules](https://nodejs.org/api/esm.html#esm_experimental_json_modules). What do you want `test.js` transformed to?

Using [simport](https://github.com/coderaiser/simport):

```js
import {createCommons} from 'simport';
const {require} = createCommons(__import.meta.url);

// and then you can use `require` in `ESM`
```

or 

```js
import "./base.json";
```

And you should use `--experimental-json-modules`

**Comment by EvgenyOrekhov** [comment]
I'm trying to understand how it's supposed to work.

I just tried adding `"type": "module"` to my `package.json`, and I still see no warnings from Putout:

![image](https://user-images.githubusercontent.com/8045060/116121771-9f871700-a6c9-11eb-8c53-26d55985d7fd.png)

**Comment by coderaiser** [comment]
The [simplest cases](https://github.com/coderaiser/putout/tree/v16.11.0/packages/plugin-convert-commonjs-to-esm) supported right now, like:

```diff
-const fs = require(‘fs’);
+import fs from ‘fs’;
```

**Comment by EvgenyOrekhov** [comment]
Oh, right, of course!

---

OK, I'm trying it, but still can't make it work. Could you please check out this PR: https://github.com/EvgenyOrekhov/eslint-config-hardcore/pull/281 and tell me why is there no warning from `convert-commonjs-to-esm`?

You can see it in CI report that there is a warning about `"use strict"`, but there is no warning about `require`:

![image](https://user-images.githubusercontent.com/8045060/116131213-c7c84300-a6d4-11eb-9346-f575f00bed6f.png)

**Comment by coderaiser** [comment]
Looks like it is related with configuration of ` convert-commonjs-to-esm`, the whole rule is disabled by default with:

```json
{
    "rules": {
        "convert-commonjs-to-esm": "off"
    }
}
```

And when you partially enable it with:

```json
{
    "rules": {
        "convert-commonjs-to-esm": "off",
        "convert-commonjs-to-esm/require": "on",
        "convert-commonjs-to-esm/exports": "on"
    }
}
```

It's do not get loaded, so one of solutions could be to enable the whole rule, and then disable parts you don't need.
Anyways it looks useful, worth fixing and landed in `@putout/engine-loader v4.3.0` 🎉 .

Is it works for you :)?

**Comment by EvgenyOrekhov** [comment]
It works now, thank you!
